### ISSUE TITLE
Tracing using Java binding submitAndWait() does not seem to work properly

### ISSUE THREAD

**Issue by judywu-da** [digital-asset/daml#15187]
# Affected Daml version

# Bug description

When looking at the traces after calling submitAndWait() in jaeger, the traces do not seems to link up properly with participants sequencers and mediators. While with submit(), the trace seems link all component properly.

# To reproduce
1. Spin up Jaeger and Ledger
2. use submitAndWait() to submit() in a Java Ledger Client against the ledger and look for the trace at com.daml.ledger.api.v1.CommandService/SubmitAndWait on Jaeger dashboard, it only contains one span. Yet while using submit(), and look for trace at com.daml.ledger.api.v1.CommandSubmissionService/Submit on Jaeger, it contains all spans.

# Expected behavior
For submitAndWait() to work like submit(), we are expecting to see all spans across participants, sequencers and mediators in its trace

# Additional context

**Comment by stefanobaghino-da** [comment]
This is not a Java bindings issue. As of now, the Ledger API server only supports tracing on a select sub-set of services, namely:

- `CommandSubmissionService`
- `ConfigManagementService`
- `PackageManagementService`
- `PartyManagementService`

I'm currently talking to @mziolekda in https://github.com/digital-asset/daml/issues/14256 to understand how they prefer for this issue to be tracked in the future.

**Comment by stefanobaghino-da** [comment]
https://github.com/digital-asset/daml/issues/14256#issuecomment-1277524323 by @mziolekda 

> I have created an epic for it on our backlog: https://digitalasset.atlassian.net/browse/DPP-1275
> It is not planned for Q4. If you feel it should be addressed urgently, a priority call must be made by Bernhard and Ratko.

**Comment by raphson93** [comment]
> [#14256 (comment)](https://github.com/digital-asset/daml/issues/14256#issuecomment-1277524323) by @mziolekda
> 
> > I have created an epic for it on our backlog: https://digitalasset.atlassian.net/browse/DPP-1275
> > It is not planned for Q4. If you feel it should be addressed urgently, a priority call must be made by Bernhard and Ratko.

Hi @stefanobaghino-da , may I know the status ?

**Comment by mziolekda** [comment]
The gap in the command service calls have been closed as part of the 2.7 release. Please see the section on "Distributed Tracing Enhancements" in the [2.7 release notes](https://blog.digitalasset.com/developers/release-notes/2.7.0).

This applies to all command service calls, i.e.
```
rpc SubmitAndWait (SubmitAndWaitRequest) returns (google.protobuf.Empty);
rpc SubmitAndWaitForTransactionId (SubmitAndWaitRequest) returns (SubmitAndWaitForTransactionIdResponse);
rpc SubmitAndWaitForTransaction (SubmitAndWaitRequest) returns (SubmitAndWaitForTransactionResponse);
rpc SubmitAndWaitForTransactionTree (SubmitAndWaitRequest) returns (SubmitAndWaitForTransactionTreeResponse);
```

**Comment by stefanobaghino-da** [comment]
@mziolekda Looks like this ticket can be closed, please do reopen it if I missed something.
